notes/2022/2022-11-03.md

```diff
@@ -0,0 +1,61 @@
+# 2022-11-03 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2022-11-03-transcript.md`](2022-11-03-transcript.md)
+
+## Attending
+
+* Brandon Mills (@btmills) - TSC
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @btmills took notes.
+
+## Topics
+
+### [chore: standardize npm script names per](https://github.com/eslint/eslint/pull/16315)
+
+**TSC Summary:** This PR reformats our npm scripts to follow the standard we have already checked in.
+
+**TSC Questions:**
+
+1. Do we want to rename the release-related scripts? We would just have to update the Jenkins job before doing the next release.
+2. Do we want to include linting in `npm test`? 
+
+* @mdjermanovic left [a comment](https://github.com/eslint/eslint/pull/16315#issuecomment-1248750725) elaborating on the second question.
+* @mdjermanovic is in favor of (1), believes updating Jenkins shouldn't be a problem, and has a question about the `build:` namespace to discuss on the PR.
+* @btmills is in favor of both.
+* @mdjermanovic prefers that `npm test` ensures the repository is in a "good state" but will defer to the others.
+* @nzakas notes that [the documentation](https://eslint.org/docs/latest/developer-guide/package-json-conventions#test) splits linting and testing.
+* @mdjermanovic supports following the standard.
+
+**Resolution:** We've agreed to 1) rename release-ralated scripts and 2) split linting and testing.
+
+### [Team Participation](https://github.com/eslint/tsc-meetings/issues/387#issuecomment-1301537844)
+
+> @nzakas: Agenda item: I’m getting concerned about the low participation rate of the team. We are getting down to the point where most of the time we only have two consistently active people (me and mdjermanovic). We are missing discussions and issues and PRs are taking longer and longer to merge. This isn’t sustainable. Let’s talk about how we can improve the situation.
+
+* All agree that we need more people actively involved.
+* @nzakas conjectures we were unclear about expectations when people joined the team and suggests requesting one hour per week/12 minutes per weekday responding to issues and PRs.
+* @mdjermanovic suggests people are unfamilier with the subject of the issue or PR.
+* @nzakas says people can repro or share an opinion on an option without full understanding, and that team members hould eventually become familiar with as much of the project as possible.
```

Comment:
By the way, I use an IDE extension to avoid spelling mistakes: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker